### PR TITLE
Refresh README and make fixtures more discoverable

### PR DESCRIPTION
**Pull request by danyf90**
Resolves https://github.com/tuist/tuist/issues/YYY
Request for comments document (if applies):

### Short description 📝

> Describe here the purpose of your PR.

### How to test the changes locally 🧐

> Include a set of steps for the reviewer to test the changes locally.

### Checklist ✅

- [x] The code architecture and patterns are consistent with the rest of the codebase.
- [x] The changes have been tested following the [documented guidelines](https://docs.tuist.io/contributors/testing-strategy/).
- [x] The PR includes the label `changelog:added`, `changelog:fixed`, or `changelog:changed`, whenever it should be included in the “Added”, “Fixed” or “Changed” section of the CHANGELOG. Note: when included in the CHANGELOG, the title of the PR will be used as entry, please make sure it is clear and suitable.
- [x] In case the PR introduces changes that affect users, the documentation has been updated.
- [x] In case the PR introduces changes that affect how the cache artifact is generated starting from the `TuistGraph.Target`, the `Constants.cacheVersion` has been updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,10 @@
   <img src="assets/header.gif"/><br>
   <img src="https://github.com/tuist/tuist/workflows/Tuist/badge.svg" alt="Tuist">
   <img src="https://img.shields.io/github/v/release/tuist/tuist?include_prereleases&style=flat-square" alt="Latest Version">
-  <a href="http://community.tuist.io"><img src="https://img.shields.io/badge/community-tuist-blue?style=flat-square" alt="Community"></a>
```
Comment (danyf90) [inline README.md:5]: this was broken

File: README.md [hunk 2]
```diff
@@ -2,13 +2,10 @@
...
-  <img src="https://img.shields.io/github/forks/tuist/tuist?style=flat-square" alt="GitHub forks">
-  <img src="https://img.shields.io/github/stars/tuist/tuist?style=flat-square" alt="GitHub stars">
   <img src="https://img.shields.io/github/commit-activity/w/tuist/tuist?style=flat-square" alt="Commit Activity">
-    <img src="https://img.shields.io/github/contributors/tuist/tuist?style=flat-square" alt="Contributors">
```
Comment (danyf90) [inline README.md:9]: These are redundant as already part of GitHub UI

File: README.md [hunk 3]
```diff
@@ -2,13 +2,10 @@
...
   <img src="https://img.shields.io/github/commits-since/tuist/tuist/latest?style=flat-square" alt="Latest Commits">
   <a href="http://twitter.com/tuistio"><img src="https://img.shields.io/twitter/follow/tuistio?style=flat-square" alt="Twitter Followers"></a>
+  <a href="https://join.slack.com/t/tuistapp/shared_invite/zt-g38gajhj-D6LLakrPnVCy4sLm24KxaQ"><img src="https://img.shields.io/badge/slack-tuist-green.svg?logo=slack" alt="Slack Workspace">
```
Comment (danyf90) [inline README.md:8]: There was no link to slack in the Readme, should we make it even more relevant (e.g. mentioning it somewhere below)?

File: projects/docs/docs/examples/fixtures.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: Fixtures
+slug: '/examples/fixtures'
+description: 'This page documents where to find Tuist fixtures.'
+---
+
+### Fixtures
```
Comment (luispadron) [inline projects/docs/docs/examples/fixtures.md:7]: This is a great addition

**Review by luispadron — APPROVED** [review]

**Pull request merged**